wordpress-mobile/WordPress-Android
# Centering vertically the items of reader bottom sheet.

This PR is related to issue 3 found during #11343 review (see [here](https://github.com/wordpress-mobile/WordPress-Android/pull/11343#pullrequestreview-361948620) for more context)

### To test
- Open the Reader bottom sheet both in SITES and TAGS pages and check that the items are vertically aligned (here below two images for reference)

| ![Screenshot_1582244689](https://user-images.githubusercontent.com/47797566/74992637-fac7d480-5448-11ea-8437-475771cd732f.png) | ![Screenshot_1582244714](https://user-images.githubusercontent.com/47797566/74992672-18953980-5449-11ea-9bc6-35788abafd89.png) |
|---|---|

PR submission checklist:
